scripts/build-app.js

```diff
@@ -66,10 +66,13 @@ function fixHtmlSubresourceUrls(result, buildOptions) {
   fs.writeFileSync(htmlDistPath, newHtmlText);
 
   // Rewrite the CSS, making sure we don't source icons relative to the chunks/css file.
-  const newCssText = fs
-    .readFileSync(cssDistPath, 'utf-8')
-    .replaceAll(`url(./assets`, `url(../assets`);
-  fs.writeFileSync(cssDistPath, newCssText);
+  const newCssText = fs.readFileSync(cssDistPath, 'utf-8');
+  console.log('newCssText', newCssText);
+  console.log('typeof newCssText', typeof newCssText);
+  console.log('typeof newCssText.replaceAll', typeof newCssText.replaceAll);
+  const regex = /url\(.\assets/g;
+  const next = newCssText.replace(regex, `url(../assets`);
+  fs.writeFileSync(cssDistPath, next);
```

Comment:
I had to do all of this because I'd otherwise get

```
$  yarn build
yarn run v1.22.19
$ lerna run --scope @lhci/server --scope @lhci/viewer build
lerna notice cli v3.22.0
lerna notice filter including ["@lhci/server","@lhci/viewer"]
lerna info filter [ '@lhci/server', '@lhci/viewer' ]
lerna info Executing command in 2 packages: "yarn run build"
lerna ERR! yarn run build exited 1 in '@lhci/viewer'
lerna ERR! yarn run build stdout:
$ node ../../scripts/build-app.js build ./src/ui/index.html ./dist ./
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
TypeError: fs.readFileSync(...).replaceAll is not a function
    at fixHtmlSubresourceUrls (/home/hamir/lighthouse-ci/scripts/build-app.js:71:6)
    at main (/home/hamir/lighthouse-ci/scripts/build-app.js:112:3)
error Command failed with exit code 1.
```

Reply:
```
$  nvm ls
->     v14.21.3
```